app/posts/manage-teacher-training-applications/2020-12-18-search-by-candidate-name-or-reference.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Search by candidate name or reference
+description: Let users search the applications list by candidate name or reference
+date: 2020-12-18
+---
+
+We introduced a search to allow users to search the applications list by candidate name or number.
+
+This change included:
+
+- creating a search field above the applications list to accept terms that are compared to a candidate’s full name or application reference
```

Comment:
Change to ‘Moving the search form’ from ‘Creating’

Reply:
Done